src/main/resources/META-INF/mods.toml

```diff
@@ -18,16 +18,16 @@ itemIcon = "farmersdelight:stove"
 configBackground = "minecraft:textures/block/bricks.png"
 
 [[dependencies.farmersdelight]]
-modId = "forge"
+modId = "neoforge"
 mandatory = true
-versionRange = "[47.1.0,)"
+versionRange = "[1,)"
 ordering = "NONE"
 side = "BOTH"
 
 [[dependencies.farmersdelight]]
 modId = "minecraft"
 mandatory = true
-versionRange = "[1.20,)"
+versionRange = "[1.20.2,)"
```

Comment:
The version range should exclude 1.20.3 in my opinion, as the chances of the mod working out of the box are very low.

Reply:
Agreed. It won't work because blocks require codecs in the registry.